src/tests/AabbTest.cpp

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+#include <Corrade/TestSuite/Compare/Numeric.h>
+#include <Corrade/TestSuite/Tester.h>
+#include <Magnum/Math/FunctionsBatch.h>
+#include "esp/core/Utility.h"
+#include "esp/geo/CoordinateFrame.h"
+#include "esp/geo/OBB.h"
+#include "esp/geo/geo.h"
+
+// TODO: refactor the GeoTest to use Cr::TestSuite
```

Comment:
I am testing a function in the geo module. Ideally, it should have one GeoTest, not 2 files. 

I am planning to migrate the current GeoTest to here but it would be a lot of work and beyond the scope of this diff. So will do it later.

After it is done, will rename this file to “GeoTest” and deprecate the old one.